lib/inspec/reporters/automate.rb

```diff
@@ -0,0 +1,64 @@
+# encoding: utf-8
+
+require 'json'
+require 'net/http'
+
+module Inspec::Reporters
+  class Automate < Json
+    def enriched_report
+      # grab the report from the parent class
+      final_report = report
+
+      # Label this content as an inspec_report
+      final_report[:type] = 'inspec_report'
+
+      final_report[:end_time] = Time.now.utc.strftime('%FT%TZ')
+      final_report[:node_uuid] = @config['node_uuid'] || @run_data[:platform][:uuid]
+      final_report[:report_uuid] = uuid_from_string(final_report[:end_time] + final_report[:node_uuid])
+
+      # optional json-config passthrough options
+      %w{node_name environment roles recipies}.each do |option|
+        final_report[option.to_sym] = @config[option] unless @config[option].nil?
+      end
+      final_report
+    end
+
+    def send_report
+      headers = { 'Content-Type' => 'application/json' }
+      headers['x-data-collector-token'] = @config['token']
+      headers['x-data-collector-auth'] = 'version=1.0'
+
+      uri = URI(@config['url'])
+      req = Net::HTTP::Post.new(uri.path, headers)
+      req.body = enriched_report.to_json
+      begin
+        Inspec::Log.debug "Posting report to Chef Automate: #{uri.path}"
+        Net::HTTP.start(uri.hostname,
+                        uri.port,
+                        use_ssl: uri.scheme == 'https',
+                        verify_mode: OpenSSL::SSL::VERIFY_NONE) do |http|
```

Comment:
Can we make this a config option? I would like to go to a world where we verify it